bindings/nodejs/src/nodejs_idle.cpp

```diff
@@ -132,5 +130,6 @@ void NodeJSIdle::InvokeCallbacks()
 
 void NodeJSIdle::OnIdle(uv_async_t* handle)
 {
+    (void)isolate;
```

Comment:
?? Did the compiler complain about an unused parameter? I'm confused because the only parameter in this functions is `handle`, not `isolate`.